### ISSUE TITLE
release legacy versions of the app

### ISSUE THREAD

**Issue by jimmac** [flathub/org.blender.Blender#58]
As Blender is being used for long term projects, the ability of flatpak to allow running multiple versions of the same app can truly be useful here. Unlike the typical stable/nightly scenario, it is very desirable to have older versions of the app around and allow concurrent installs and execution. Features get obsoleted, components replaced, so some projects actually rely on a particular version to be used to continue working.

It would be nice to figure out how roll out org.blender.Blender280 to be able to allow for a studio to open and tweak projects from 2019/2020 while making use of the new stuff from 2.90 going forward.

**Comment by hfiguiere** [comment]
A branch could be made for that eventually so you could have `org.blender.Blender//2.80`. (I don't have the permissions to create said branch)

**Comment by eszlari** [comment]
I was planning to submit it under a new app ID (`org.blender.Blender.LTS`) to Flathub.

**Comment by hfiguiere** [comment]
I'd find it better to have a branch for it.

**Comment by eszlari** [comment]
A branch wouldn't satisfy @jimmac's request:

> it is very desirable to have older versions of the app around and allow concurrent installs and execution.

**Comment by hfiguiere** [comment]
Yes it would. You can have the same app from different branches installed. No probem.

**Comment by eszlari** [comment]
Read the whole quote. You can't run them at the same time and they would share the same configs.

**Comment by eszlari** [comment]
> and they would share the same configs.

Forget the last part, blender configs are versioned:

```
Writing userprefs: '/home/peter/.var/app/org.blender.Blender/config/blender/2.90/config/userpref.blend' ok
```

**Comment by plafue** [comment]
I asked for some input from the flatpak organisation. Moving forward the __latest__ LTS version will be maintained next to the last released version. I know this does not quite cover the request of this ticket (older versions) but it will be possible to use a stable LTS version from now on.

The current LTS version (2.93) will be released under its own application name `BlenderLTS` or similar. Will keep you posted.

Thanks to @barthalion for the input.

**Comment by Vitorvlv** [comment]
Any updates on this?

**Comment by mbridon** [comment]
Hi there, I'm @bochecha but had to chage my Github account to @mbridon :confused: 

I maintain the official Flatpak build of Blander, so I guess I could do the sae for a BlenderLTS.

What version would that be and how can I follow new LTS releases of Blender ?

**Comment by jimmac** [comment]
The LTS releases are being published at https://www.blender.org/download/lts/

**Comment by mbridon** [comment]
bot, build org.blender.BlenderLTS

**Comment by hfiguiere** [comment]
Someone can always pick it up where it was left.

**Comment by mbridon** [comment]
Hi, can some admin unsubscribe @bochecha from the "Blender on flathub" issues?

I can't log into that account anymore, and haven't been able to do so for years now, after I got sick and had to give up on maintaining that flatpak build, and all I'm getting now are these lousy notifications...

Github themselves are unable to do anything for me, please, can someone remove @bochecha from this? 😢
